spec/models/program_session_spec.rb

```diff
@@ -1,5 +1,145 @@
 require 'rails_helper'
 
-RSpec.describe ProgramSession, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe ProgramSession do
+
+  let(:proposal) { create(:proposal_with_track, :with_two_speakers) }
+
+  it "responds to .create_from_proposal" do
+    expect(ProgramSession).to respond_to(:create_from_proposal)
+  end
+
+  describe ".create_from_proposal" do
```

Comment:
Make sure to use the form #create_from_proposal with the #. That's a ruby convention (like in routes.rb when you use a string to specify the action a route maps to, it's 'controller#action'